ui/pages/settings/security-tab/security-tab.component.js

```diff
@@ -42,7 +43,7 @@ export default class SecurityTab extends PureComponent {
     history: PropTypes.object,
     participateInMetaMetrics: PropTypes.bool.isRequired,
     setParticipateInMetaMetrics: PropTypes.func.isRequired,
-    showIncomingTransactions: PropTypes.bool.isRequired,
+    showIncomingTransactions: PropTypes.bool,
```

Comment:
should this stay as required?

Reply:
It's technically not required, since in `security-tab.container.js` it has this default value:
`featureFlags: { showIncomingTransactions } = {},`

I changed this `isRequired` so I could write a new test and get 100% test coverage on `security-tab.test.js`

Now one could argue that's maybe the wrong decision... that the right thing would be to remove the default value and leave it as `isRequired`.  So I guess let's discuss that.